ckanext/iati/plugins.py

```diff
@@ -146,28 +146,28 @@ def _modify_group_schema(self, schema):
         _unicode_safe = p.toolkit.get_validator('unicode_safe')
         _group_name_validator = p.toolkit.get_validator('group_name_validator')
 
-        default_validators = [_ignore_missing, _convert_to_extras, unicode]
+        default_validators = [_ignore_missing, _convert_to_extras, str]
```

Comment:
This is invalid, the validator can't be a builtin. There's another validator to replace unicode, I think it's `valid_unicode`